src/main/java/com/danielagapov/spawn/Services/User/IUserService.java

```diff
@@ -11,4 +12,5 @@ public interface IUserService {
     public List<UserDTO> getUsersByTagId(UUID tagId);
     public UserDTO saveUser(UserDTO user);
     public UserDTO replaceUser(UserDTO user, UUID id);
+    public FriendRequests saveFriendRequest(FriendRequests friendRequest);
```

Comment:
Please make a DTO, as discussed on Slack, that can simply inherit (`extends`) from `FriendRequests` for now.